src/common/config.cpp

```diff
@@ -11,10 +12,76 @@ namespace marian {
 
 size_t Config::seed = (size_t)time(0);
 
+Config::Config(const std::string options,
+               cli::mode mode /*= cli::mode::training*/,
```

Comment:
What is this mode for? How is a command-line parser modal? Add a comment

Reply:
Added TODO